google/cloud/forseti/common/data_access/csv_writer.py

```diff
@@ -477,6 +478,24 @@
 }
 
 
+def normalize_nested_dicts(row):
+    """Transform nested dicts into json strings with sorted keys.
+
+    Args:
+        row (dict): A dictionary to normalize.
+
+    Returns:
+        dict: A row with nested dicts transformed to json string.
+    """
+    new_row = {}
+    for key, value in row.items():
+        if isinstance(value, dict):
+            new_row[key] = json.dumps(value, sort_keys=True)
+        else:
+            new_row[key] = value
```

Comment:
I think the newly created dict might have different order when being called multiple times with the same input, does it affect the comparison? 

e.g. 
input: {a: 1, b:2}
output can either be {b:2, a:1} or {a:1, b:2}